sonarqube/provider.go

```diff
@@ -67,7 +75,13 @@ type ProviderConfiguration struct {
 }
 
 func configureProvider(d *schema.ResourceData) (interface{}, error) {
+	transport := cleanhttp.DefaultPooledTransport()
```

Comment:
this is the default transport that you get with an http client created via `retryablehttp.NewClient()`